prometheus-rules/prometheus-vmware-rules/alerts/nsxt.alerts

```diff
@@ -229,7 +229,7 @@ groups:
     expr: vrops_nsxt_mgmt_node_connectivity_status{state="DISCONNECTED"}
     for: 5m
     labels:
-      severity: info
+      severity: warning
       tier: vmware
       service: network
       support_group: compute
```

Comment:
@atchyu, please ensure that the playbook contains the correct documentation path and update the alert definition accordingly.